spacecmd/src/spacecmd/system.py

```diff
@@ -157,6 +157,8 @@ def manipulate_child_channels(self, args, remove=False):
                                             system_id,
                                             child_channels)
 
+    return
```

Comment:
Shouldn't this be a `return None` like the other explicit `return None` statements that you've added?

Reply:
Pylint didn't complain about it :)
Reason for the other explicit `return None`s is, that other `return` statements in the **same** function return a value. So the pylint statement is: Either none or all return statements return should a value (within the same function).

Reply:
I see, I didn't expand any of the views and I don't (yet) know all the pylint expectations. But if pylint does not complain, then I won't complain either :)